### ISSUE TITLE
ansible-lint doesn't detect missing 'dest' for module 'copy'

### ISSUE THREAD

**Issue by hunter86bg** [ansible-community/ansible-lint#627]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
[ansible@ansible ~]$ ansible --version
ansible 2.9.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/ansible/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Aug  7 2019, 00:51:29) [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
[ansible@ansible ~]$ ansible-lint --version
ansible-lint 4.1.0
```

- ansible installation method: rpm from ansible site
- ansible-lint installation method:  pip

# Desired Behaviour
ansible-lint must check for mandatory requisites in the 'copy' module .

# Actual Behaviour (Bug report only)
ansible-lint doesn't complain for the following:
```
[ansible@ansible ~]$ cat deploy-facts.yml
---
- name: "Deploy Custom facts"
  hosts: all
  vars:
    remote_dir: /etc/ansible/facts.d
    facts_file: lesson5/facts/custom_facts.fact
  tasks:
    - name: "Create remote facts directory"
      file:
        path: "{{ remote_dir }}"
        state: directory
        recurse: True
    - name: "Copy the facts file"
      copy:
        src: "{{ facts_file }}"
        dst: "{{ remote_dir }}"
[ansible@ansible ~]$ ansible-lint deploy-facts.yml
[ansible@ansible ~]$ ansible-playbook --syntax-check deploy-facts.yml

playbook: deploy-facts.yml
[ansible@ansible ~]$ ansible-playbook deploy-facts.yml

PLAY [Deploy Custom facts] **********************************************************************************************************************************

TASK [Create remote facts directory] ************************************************************************************************************************
ok: [debian]
ok: [centos7]

TASK [Copy the facts file] **********************************************************************************************************************************
fatal: [debian]: FAILED! => {"changed": false, "msg": "dest is required"}
fatal: [centos7]: FAILED! => {"changed": false, "msg": "dest is required"}

TASK [Create remote facts directory] ************************************************************************************************************************
ok: [ubuntu18]

TASK [Copy the facts file] **********************************************************************************************************************************
fatal: [ubuntu18]: FAILED! => {"changed": false, "msg": "dest is required"}

PLAY RECAP **************************************************************************************************************************************************
centos7                    : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
debian                     : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
ubuntu18                   : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

[ansible@ansible ~]$ sed -i 's/dst/dest/g' deploy-facts.yml
[ansible@ansible ~]$ ansible-playbook deploy-facts.yml

PLAY [Deploy Custom facts] **********************************************************************************************************************************

TASK [Create remote facts directory] ************************************************************************************************************************
ok: [debian]
ok: [centos7]
ok: [ubuntu18]

TASK [Copy the facts file] **********************************************************************************************************************************
changed: [debian]
changed: [centos7]
changed: [ubuntu18]

PLAY RECAP **************************************************************************************************************************************************
centos7                    : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
debian                     : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
ubuntu18                   : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

**Comment by ssbarnea** [comment]
As we started to make use of syntax check, this counts as an upstream bug https://github.com/ansible/ansible/issues/73338
